content/en/integrations/_index.md

```diff
@@ -95,6 +95,23 @@ cascade:
     path: /integrations/feed.md
   aliases:
     - /integrations/rss/
+- _target:
+    path: /integrations/java.md
+  aliases:
+    - /agent/faq/jmx_integrations/
```

Comment:
This one doesn't seem to work as expected

Reply:
The `jmx_integrations` one is from the ticket. It won't work until I merge the Dogweb one - the existence of an alias over there is causing a conflict. I'll make sure it works before I merge though 👍